controller_manager:
  ros__parameters:
    update_rate: 100 # hz tutto
    state_broadcaster:
      type: pi3hat_state_broadcaster/Pi3Hat_State_Broadcaster # stampa roba su topic 
    joint_controller:
      type: pi3hat_joint_group_controller/Pi3Hat_Joint_Group_Controller # prende messaggi da topic 
    OM_controller:
      type: pi3hat_vel_controller/Pi3Hat_Vel_Controller
state_broadcaster:
  ros__parameters:
    joints:
      - "RF_HFE"
      - "RF_KFE"
      - "LF_HFE"
      - "LF_KFE"
      - "LH_HFE"
      - "LH_KFE"
      - "RH_HFE"
      - "RH_KFE"
      - "RF_WHEEL"
      - "LF_WHEEL"
      - "LH_WHEEL"
      - "RH_WHEEL"

    performance_index: true
OM_controller:
  ros__parameters:
    homing_duration: 10.0
    input_frequency: 100
    mecanum_angle: 135.0
    
    


